referencePose:
  - 0.6757325532261766
  - -0.09445390849921459
  - 0.2367249188577119
  - -3.101000444582022
  - 0.04503867342607275
  - -0.3652856320668224
jointsValues:
  - 1.661785617885164
  - -1.194629545537961
  - -1.405430810339032
  - -2.139007271683007
  - -1.127202390081327
  - 1.644143418974346
  - 1.819293590007557